mkosi.build

```diff
@@ -25,3 +25,11 @@ make -j `nproc`
 
 make check
 make install
+
+mkdir -p $DESTDIR/etc
+
+cat > $DESTDIR/etc/issue <<EOF
+\S (built from systemd tree)
```

Comment:
hmm, can this work? I mean, this is being run in the build environment, not on the host. We hence are not working on the original git tree, hence we shouldn't be able to see that, no?

I figure we could make mkosi pass the git tag always as an env var to the build script though. I think that could be quite useful for most cases. will work on that